node-proxy/lib/utils/http-utils.js

```diff
@@ -29,12 +29,12 @@ exports.createProtocolServer = function(protocol, opts) {
       break;
     case 'https':
       var ssl_opts  = {
-        secureProtocol: 'SSLv23_method',
         secureOptions: constants.SSL_OP_NO_SSLv3
       };
       ssl_opts.ca   = fs.readFileSync(opts.ca);
       ssl_opts.cert = fs.readFileSync(opts.certificate);
       ssl_opts.key  = fs.readFileSync(opts.private_key);
+      ssl_opts.ciphers = opts.ciphers;
```

Comment:
I worried about that a bit when putting this together. None of the other options, when used, account for the option not being present in the configuration file. I can add some handling so that the a default cipher suite is used if you deem it necessary.